requirements-tpu.txt

```diff
@@ -4,4 +4,4 @@
 # Dependencies for TPU
 # Currently, the TPU backend uses a nightly version of PyTorch XLA.
 # You can install the dependencies in Dockerfile.tpu.
-ray
+ray[default,serve]
```

Comment:
Please remove Ray Serve from the dependency. vLLM does not need to be used with Ray Serve, and we'd like to minimize the dependencies.

Reply:
Can we still install ray[default]? The reason for this is that the GCS endpoint needs to run through the Ray dashboard which does not get installed if you just do `pip install ray`. The endpoint is needed in order for other Ray workers to join the cluster for multihost inference.

Reply:
Oh, how is `ray[default]` different from just `ray`?

Reply:
According to https://docs.ray.io/en/latest/ray-overview/installation.html, `ray[default]` includes the Ray dashboard while `ray` is just the Ray core libraries.

The reason for including the dashboard (in addition to debuggability) is that the GCS and Ray Job endpoints are exposed through the dashboard. So without it, the other Ray nodes are not able to join the cluster. For example the Kubernetes operator initializes Ray worker nodes by having them ping the GCS endpoint.